lib/cartopy/feature.py

```diff
@@ -158,18 +242,18 @@ class NaturalEarthFeature(Feature):
     See http://www.naturalearthdata.com/
 
     """
-    def __init__(self, category, name, scale, **kwargs):
+    def __init__(self, category, name, scaler, **kwargs):
```

Comment:
Let's keep the kwarg as ``scale`` to avoid breaking API without needing too.

Reply:
Agreed :+1: